addons/knowledge/static/src/js/field_html.js

```diff
@@ -63,4 +66,20 @@ FieldHtml.include({
         }
         return options;
     },
+    /**
+     * When the user clicks on an article link, we can directly open the
+     * article in the current view without having to reload the page.
+     * @param {Event} event
+     */
+    _onLinkClick: function (event) {
+        const href = $(event.currentTarget).attr('href');
+        const matches = href.match(/^\/article\/(\d+)(?:\/|(?:#|\?).*)?$/);
```

Comment:
@julien-banken nope ! 
We need to use classes to target an article link.
matching url is broken by design (same remark than 2 month ago)